docs/client/Account.md

```diff
@@ -0,0 +1,15 @@
+# Account
+
+## Properties
+Name | Type | Description | Notes
+------------ | ------------- | ------------- | -------------
+**Description** | **string** |  | [optional] [default to null]
```

Comment:
Just checking, but is the capitalization correct on all of these properties?

Reply:
I hope so. These are coming from the Swagger Schema that we produce on Kato.
I think these values may be specified by annotation but even if they are not correct, the will match the values generated by Swagger-Codegen, so we should be OK.